plugins/BEdita/API/src/Controller/Component/PaginatorComponent.php

```diff
@@ -34,12 +34,15 @@ class PaginatorComponent extends CakePaginatorComponent
         'whitelist' => ['page', 'page_size', 'sort'],
     ];
 
+    protected $absoluteLimit = 500;
```

Comment:
Docblock is missing for this class member. 🙃 

However, I'd probably use `const MAX_LIMIT = 500;` instead.